docs/en/operation/User-CLI.md

```diff
@@ -977,6 +986,25 @@ For example, `getCapacityBytes` can be used to verify if your cluster is set up
 $ ./bin/alluxio fs getCapacityBytes
 ```
 
+### getCmdStatus
+
+The `getCmdStatus` command returns the detailed distributed command status based on a given JOB_CONTROL_ID.
+The detailed status includes:
+1. Successfully loaded or copied file paths.
+2. Statistics on the number of successful and failed file paths.
+3. Failed file paths, logged in a separate csv file.
+
+For example, `getCmdStatus` can be used to check what files are loaded in a distributed command, and how many succeeded or failed.
+
+```console
+$ ./bin/alluxio fs getCmdStatus $JOB_CONTROL_ID
+Sample Output:
```

Comment:
this example output is great, would like the same for the other distributed commands if possible

Reply:
Added more sample outputs.